src/webview/mod.rs

```diff
@@ -126,8 +126,7 @@ pub struct WebViewAttributes {
   /// different Origin headers across platforms:
   ///
   /// - macOS: `<scheme_name>://<path>` (so it will be `wry://examples` in `custom_protocol` example)
-  /// - Linux: Though it's same as macOS, there's a [bug] that Origin header in the request will be
-  /// empty. So the only way to pass the server is setting `Access-Control-Allow-Origin: *`.
+  /// - Linux: Same as macOS.
```

Comment:
just change it to `- macOS and Linux: `.

you should also add a note that for headers to work, you need to enable `linux-headers` flag

Reply:
Thanks for review! I fixed it